ocs_ci/utility/csr.py

```
@@ -134,8 +135,113 @@ def get_nodes_csr():
+        return False
+
+    fixed = False
+    for node_name, node_ip in missing_nodes:
+        try:
+            node = VSPHERENode(node_ip)
+            if node.is_kubelet_active():
+                logger.info(f"{node_name} kubelet is active, skipping ostree check")
+                continue
+
+            current_image = node.get_active_ostree_image()
+            logger.info(f"{node_name} current ostree image: {current_image}")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)